packages/outline-core-link/src/config.ts

```diff
@@ -0,0 +1,65 @@
+export type LinkTargetType = '_blank' | '_self' | '_parent' | '_top';
+export type LinkRelType =
+  | 'alternate'
+  | 'author'
+  | 'bookmark'
+  | 'external'
+  | 'help'
+  | 'license'
+  | 'next'
+  | 'nofollow'
+  | 'noopener'
+  | 'noreferrer'
+  | 'prev'
+  | 'search'
+  | 'tag';
+
+export interface LinkType {
+  linkTarget?: LinkTargetType;
+  linkRel?: LinkRelType;
+  linkText?: string;
+  linkHref?: string;
+}
```

Comment:
Should likely move to `outline-core-link.ts` file or the `index.ts` directly.

Reply:
I'm fine with leaving the pattern for component level separation of logic. It was originally there for just that, but very few components adopted the standard, or had much need for it. 

We can leave this as is now without the Storybook-isms.